### PR TITLE
DISP-3754: Add/Edit customer endpoints

### PR DESCRIPTION
**Pull request by kswin**
Add endpoints to update and add a customer. Follows the pattern of job, where addCustomer is nested under organization entity since you need an org id to create a customer.

### CODE REVIEW

**Comment by jraede** [comment]
🐑

File: lib/entities/customer.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  const customer = function(id) {
+    return {
+      update: (customerProperties) => {
+        return client
+          .doAuthenticatedRequest('PATCH', `${CUSTOMERS}/${id}`, customerProperties)
+          .then(response => response.customer);
+      },
+    };
+  };
+  return customer;
```
Comment (aldeed) [inline lib/entities/customer.js]: Could do `return function(id) {...`

File: lib/entities/organization.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  const organization = function(id) {
+    return {
+      addCustomer: (customer) => {
+        return client
+          .doAuthenticatedRequest('POST', `${CUSTOMERS}`, Object.assign({}, { organization_id: id }, customer))
+          .then(response => response.customer);
+      },
+    };
+  };
+  return organization;
```
Comment (aldeed) [inline lib/entities/organization.js]: Could do `return function(id) {...`

**Comment by aldeed** [comment]
A couple minor comments, but this should be added to the readme too

**Pull request merged**